examples/example-deno-runtime/tests.ts

```diff
@@ -267,8 +267,7 @@ const imports: Imports = {
   },
 
   log: (message: string): void => {
-    // The plugin is not expected to log anything unless it panics:
-    fail("Plugin panic: " + message);
+    console.log(message);
```

Comment:
Should we have some prefix here, so it at least becomes clear where the message is coming from?

Reply:
Yes, that makes sense.